tests/test-groups.py

```diff
@@ -30,89 +30,115 @@
 
 
 
-def test_groups_internal2():
-    d0 = dt.Frame([[1,   5,   3,   2,   1,    3,   1,   1,   None],
+def test_groups1():
+    DT = dt.Frame([[1,   5,   3,   2,   1,    3,   1,   1,   None],
                    ["a", "b", "c", "a", None, "f", "b", "h", "d"]],
                   names=["A", "B"])
-    d1 = d0[:, :, by("A")]
-    # gb = d1.internal.groupby
-    # assert gb.ngroups == 5
-    # assert gb.group_sizes == [1, 4, 1, 2, 1]
```

Comment:
btw, how to get access to number of groups in python since `.internal` is gone?

Reply:
well, that property no longer exists, so there is no way to access it.

Reply:
Yeah, but sometimes it is nice to know number of groups.

Reply:
It sure is! So maybe we can add some kind of helper function to access this and similar properties. Though I'm not sure how it's supposed to work API-wise